src/Build/Graph/ProjectInterpretation.cs

```diff
@@ -239,10 +246,89 @@ ProjectItemInstance projectReference
                         $"{setConfigurationString};{setPlatformString};{setTargetFrameworkString}").ToImmutableDictionary();
                 }
             }
-
             return new GlobalPropertyPartsForMSBuildTask(newProperties, defaultParts.AdditionalProperties, newUndefineProperties);
         }
 
+        private static string GetCompatiblePlatform(ProjectItemInstance projectReference){
+                
+                var ReferencedProject = new Project(projectReference.EvaluatedInclude);
+                string projectReferencePlatformMetadata =  ReferencedProject.GetPropertyValue(PlatformMetadataName);
+                var CurrentProjectPlatform = projectReference.Project.GetPropertyValue(PlatformMetadataName);
+                var platformLookupTable = projectReference.Project.GetPropertyValue(PlatformLookupTableMetadataName);
+                var referencedLookupTableMetadata = ReferencedProject.GetPropertyValue(PlatformLookupTableMetadataName);
+
+#nullable enable
```

Comment:
Avoid these. Either use nullable or don't (ideally you would)

Reply:
removed and fixed here